src/ui/components/SecondaryToolbox/react-devtools/components/SelectedElement.tsx

```diff
@@ -127,7 +127,7 @@ export function SelectedElement({
 
   return (
     <div className={styles.Panel} data-is-pending={isPending || undefined}>
-      <div className={styles.TopRow}>
+      <div className={styles.TopRow} data-test-id="ReactDevTools-TopRow">
```

Comment:
Seems like we don't need this test id.

```suggestion
```

Reply:
Yeah, I started to add this so we could narrow the locator selection, then decided to just put one right on the button.